shared/src/common/types/worker/worker.ts

```diff
@@ -0,0 +1,5 @@
+export { type EAMCreateWorkerRequestDto } from './eam-create-worker-request-dto.type';
+
+export { type EAMCreateWorkerResponseDto } from './eam-create-worker-response-dto.type';
+
```

Comment:
```suggestion
export { type EAMCreateWorkerResponseDto } from './eam-create-worker-response-dto.type';
```

Reply:
didn't understand the edit,  lines are the same

Reply:
just remove spaces between imports